adr/2022-08-decouple-from-ietf.md

```diff
@@ -14,16 +14,16 @@ Related Issues:
 
 ## Context and Problem Statement
 
-Currently JSON Schema follows the IETF Internet-Draft (I-D) process for spec
-development and releases but aren't associated with IETF in any way. Our
+Currently JSON Schema loosely follows the IETF Internet-Draft (I-D) process for
+spec development and releases but isn't associated with IETF in any way. Our
 perceived involvement with IETF causes confusion and misunderstanding within our
```

Comment:
It's actually common and correct for individual drafts (drafts with someone's name in the identifier) to not have any formal connection to the IETF.  You only get a formal connection when the I-D is adopted by a working group.  So this is not actually unusual or incorrect.  People thinking there is IETF support behind a draft-handrews or draft-bhutton is incorrect.  (sorry I didn't notice this last time)

Reply:
Yes, that was what I was trying to express. I'll see if I can make that more clear.

Reply:
If it's common for individual drafts to not have any formal connection with IETF, why are they saying that involvement with them would basically require starting over with JSON Schema?
It sounds like there were technical concerns. What were the issues?

Reply:
> why are they saying that involvement with them would basically require starting over with JSON Schema?
It sounds like there were technical concerns. What were the issues?

@devinbost, that's a good question that maybe @handrews can provide more details about, but I'd prefer not to get into it in this PR. We don't expect that joining an IETF working group would mean starting over and the prospect was never a factor in this decision. If we were to join an IETF working group it would not be the one we talked to in the past, it would be the HTTPAPIs working group that we have a good relationship with today.